consensus/src/network_interface.rs

```diff
@@ -64,6 +65,23 @@ pub enum ConsensusMsg {
     CommitDecisionMsg(Box<CommitDecision>),
 }
 
+impl Display for ConsensusMsg {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
```

Comment:
we probably don't want this in Display, this will make all the debug print omit the actual message content

Reply:
We are not using Display anywhere for this for now but sure changed it.